test_scripts/API/Restructuring_OnResetTimeout/004_BC_OnResetTimeout_HMI_did_not_respond_notification_without_resetPeriod.lua

```diff
@@ -0,0 +1,48 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0189-Restructuring-OnResetTimeout.md
+--
+-- Description:
+-- In case:
+-- 1) RPC is requested
+-- 2) Some time after receiving RPC request on HMI is passed
+-- 3) HMI sends BC.OnResetTimeout(without resetPeriod) to SDL in 2 seconds after HMI request receiving
+-- 4) HMI does not send response
+-- SDL does:
+-- 1) Respond with GENERIC_ERROR resultCode to mobile app in 12 seconds after mobile request receiving
```

Comment:
it says `in 12 seconds`, but I see 17000 in code.
Please make sure the description and the code consistent.